lib/kennel/models/slo.rb

```diff
@@ -16,9 +17,18 @@ class Slo < Record
         tags: -> { @project.tags },
         query: -> { DEFAULTS.fetch(:query) },
         description: -> { DEFAULTS.fetch(:description) },
-        monitor_ids: -> { DEFAULTS.fetch(:monitor_ids) }
+        monitor_ids: -> { DEFAULTS.fetch(:monitor_ids) },
+        thresholds: -> { DEFAULTS.fetch(:thresholds) }
       )
 
+      def initialize(*)
+        super
+
+        raise ValidationError, "Threshold warning must be greater-than critical value" if thresholds.any? do |t|
+          t.key?(:warning) && !t[:warning].nil? && t[:warning].to_f <= t[:critical].to_f
```

Comment:
```suggestion
          t[:warning] && t[:warning].to_f <= t[:critical].to_f
```

Reply:
and then best make this a `if foo -newline-` multiline trailing ifs are kinda funky